src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -110,10 +173,16 @@ var Marathon = React.createClass({
       error: function () {
         this.setState({fetchState: States.STATE_ERROR});
       }.bind(this),
-      reset: true,
       success: function () {
+        var state = this.state;
         this.fetchDeployments();
-        this.setState({fetchState: States.STATE_SUCCESS});
+        var activeApp = (state.activeAppId) ?
```

Comment:
And same here right? Collection is initialised, so this will do: `var app = this.state.collection.get("/" + state.activeAppId);`, or am I wrong?